src/tlo/methods/hiv.py

```diff
@@ -746,7 +751,8 @@ def split_into_vl_and_notvl(all_idx, prob):
         # all those on ART need to have event scheduled for continuation/cessation of treatment
         # this window is 1-90 days (3-monthly prescribing)
         for person in art_idx:
-            days = self.rng.randint(low=1, high=90, dtype=np.int64)
+
+            days = self.rng.randint(low=1, high=self.parameters['dispensation_period_months']*30.5, dtype=np.int64)
```

Comment:
@tdm32  -- do we want this? 
I would have thought a refill interval of 1 day doesn't make sense.
Drawing from a uniform with this specification, would give a mean of 45 days.

I think we should use this instead, let me know what you think

```
            days = self.rng.randint(low=1, high=self.parameters['dispensation_period_months']*30.5, dtype=np.int64)
            self.sim.schedule_event(
                Hiv_DecisionToContinueTreatment(person_id=person, module=self),
                self.sim.date + DateOffset(months=self.parameters['dispensation_period_months'], days=self.rng.randint(-14,14),
            )
```

Reply:
My thinking was that at baseline, we don't know how long a person has been on ART. They may have had their last dispensation anywhere between 1-90 days ago so I use a uniform distribution to equally distribute Hiv_DecisionToContinueTreatment across the next three months.

I think it probably won't make a big difference - as long as we don't clump them within a few days of each other then I think that's fine.

Reply:
Ah yes, you're quite right. I was looking at just the code snippet and didn't notice that it was in the initiation section.